twake/backend/node/src/services/workspaces/entities/workspace_counters.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity } from "../../../core/platform/services/database/services/orm/decorators";
+import { CounterEntity } from "../../../utils/counters";
+
+export const TYPE = "workspace_counters";
+
+@Entity(TYPE, {
+  primaryKey: [["id"], "counter_type"],
+  type: TYPE,
+})
+export class WorkspaceCounterEntity extends CounterEntity {
+  @Column("id", "timeuuid")
+  id: string;
+
+  @Column("counter_type", "string")
+  counter_type: string;
+}
```

Comment:
I don't understand why we don't have the "value" field in the counter entities ?

Reply:
Nevermind I just saw the `extends CounterEntity`, this is great 👍